docs/api.md

```diff
@@ -18,18 +18,22 @@ DoT: STATION SIZE AND COLOR MUST BE IDENTICAL TO DISCLOSURE SIZE AND COLOR!
 
 ## Properties
 
-| Property           | Attribute          | Type     | Description                      |
-|--------------------|--------------------|----------|----------------------------------|
-| `arrivalStation`   | `arrivalStation`   | `String` | Station of arrival, ex: `SEA`    |
-| `arrivalTime`      | `arrivalTime`      | `String` | Time of arrival, ex: `9:06 pm`   |
-| `departureStation` | `departureStation` | `String` | Station of departure, ex: `PVD`  |
-| `departureTime`    | `departureTime`    | `String` | Time of departure, ex: `5:36 am` |
+| Property              | Attribute             | Type     | Description                              |
+|-----------------------|-----------------------|----------|------------------------------------------|
+| `arrivalStation`      | `arrivalStation`      | `String` | Station of arrival, ex: `SEA`            |
+| `arrivalTime`         | `arrivalTime`         | `String` | Time of arrival, ex: `9:06 pm`           |
+| `departureStation`    | `departureStation`    | `String` | Station of departure, ex: `PVD`          |
+| `departureTime`       | `departureTime`       | `String` | Time of departure, ex: `5:36 am`         |
+| `newArrivalStation`   | `newArrivalStation`   | `String` | Station of rerouted arrival, ex: `AVP`   |
```

Comment:
Just catching this now, but we are not using `ex` for example. Please update to `e.g.`